src/workload_manager.py

```diff
@@ -248,7 +175,7 @@ def update_config(self, manifest: ConfigFileSystemState) -> None:
         logger.debug("applying config changes")
         manifest.apply(self._container)
 
-        # Validate with amtool and raise if bad
+        # Validate and raise if bad
         try:
             self.check_config()
         except WorkloadManagerError as e:
```

Comment:
this is not true though. in the action, we set failed status if stdout is empty and stderr is nonempty. Should we raise here on those conditions too? Or (probably a bit more general): raise a custom exception from check_config, store stderr, stdout in it, and pull them out again to set the action results